testing/kuttl/e2e/cluster-start/01-psql-connect.yaml

```diff
@@ -1,10 +1,26 @@
-apiVersion: kuttl.dev/v1beta1
-kind: TestStep
-commands:
-- script: CLUSTER=cluster-start $HOME/postgres-operator/testing/kuttl/script/port-forward.sh
-  background: true
-# this sleep is needed when running in cloud environments to ensure connection-psql.sh
-# will not fail when attempting the port-forward and has suffient time to connect, may 
-# not be needed when running locally
-- script: sleep 10
-- script: CLUSTER=cluster-start $HOME/postgres-operator/testing/kuttl/script/connect-psql.sh
\ No newline at end of file
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: psql-connect
+spec:
+  template:
+    spec:
+      restartPolicy: "OnFailure"
+      containers:
+        - name: psql
+          image: registry.developers.crunchydata.com/crunchydata/crunchy-postgres:centos8-14.1-1
+          command:
+            - "bash"
+            - "-c"
+            - "psql -c 'select version();'"
```

Comment:
What do you think about a bash command to get the version and compare it against the version we expect? Unnecessary and overcomplicated? (That's what I'm beginning to think, but just want to have the discussion.)

Reply:
We can for sure add checks like that. In this case, I don't think it is necessary. If the command can't connect or can't run `select version();` it will fail and the job will fail.